source/jormungandr/jormungandr/interfaces/v1/serializer/fields.py

```diff
@@ -100,8 +100,9 @@ class ErrorSerializer(PbNestedSerializer):
 
 
 class CoordSerializer(serpy.Serializer):
-    lon = DoubleToStringField()
-    lat = DoubleToStringField()
+    lon = DoubleToStringField(display_none=True)
```

Comment:
`required=True` no?

Reply:
if 'display_none' isn't set, it will be False by default (jsonschema/fields) and 'required' will always be set to False (serpy/fields.py) if 'display_none' is False.

Reply:
I forgot to mention that 'required' is set to True by default (serpy/fields.py). So, as long as 'display_none' is set to True, 'required' is also set True